specification/dimensions/commitmentutilization.md

```diff
@@ -0,0 +1,37 @@
+# Commitment Utilization
+
+Commitment Utilization indicates whether a charge utilized a commitment during the charge period. Commitment Utilization applies to all commitments and commitment types applied to the current charge. This includes but is not limited to capacity commitments and commitment-based discounts represented by the Commitment Discount ID. Commitment Utilization is commonly used to identify wastage when a commitment has not been used and to calculate commitment utilization.
```

Comment:
Do we have definition of charge period? Maybe we should say billing period instead?

Reply:
See the ChargePeriodStart/End columns. The Charge Period is the granularity of the data, so a day or hour of usage in this case. Whatever the row is covering.

Reply:
Clear now, thanks